src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
+@import "../styles/header.scss";
+@import "../styles/card.scss";
+@import "../styles/stars.scss";
+
 body {
```

Comment:
There is no need to use 'display: block;' for the 'body'. It's block by default.